docker-compose.yml

```diff
@@ -42,3 +42,10 @@ services:
 # Make the custom-user-federation-example extension available to Keycloak. The :z option is required and tells Docker that the volume content will be shared between containers.
     - ./custom-user-federation-example/build/libs/custom-user-federation-example.jar:/opt/jboss/keycloak/standalone/deployments/custom-user-federation-example.jar:z
 
+  app:
```

Comment:
I imagine you were working with something locally here.  This will need to be removed before this can be merged.

Reply:
Fixed in: https://github.com/mrparkers/terraform-provider-keycloak/pull/763/commits/2e23e2382f1b41b0478925798c2d7a8f24e56a43